client/src/pages/Home.js

```diff
@@ -39,7 +43,7 @@ function Home() {
               key={list._id}
               list={list}
               href={`/home/${list._id}`}
-              onDelete={setDeleterOverlay}
+              onDeleteConfirm={handleDelete}
```

Comment:
Think about this: (your other components will be more simple).

```suggestion
              onDeleteConfirm={() => handleDelete(list._id)}
```

Reply:
Tried it before, and it wouldn't close my overlay, but now it's working :/ Thanks for the input, I already changed it and shortend my components